README.md

```diff
@@ -27,14 +27,14 @@ For more information about API credentials see [here](https://dev.bandwidth.com/
 # Environmental Variables
 The sample app uses the below environmental variables.
 ```java
```

Comment:
maybe change this to sh so `private` and `public` dont show up

Reply:
oops missed this one